vkconfig/README.md

```diff
@@ -39,7 +39,7 @@ Vulkan Configurator is a graphical user interface (GUI) that may be launched fro
 
 The tool is distributed differently, depending on the platform:
 - Ubuntu packages: Upon installing the `lunarg-vkconfig` package, the tools will be available from the command line as `vkconfig`.
-- Linux tarball: The vkconfig executable comes pre-built. Run vkconfig on the command line. 
+- Linux tarball: The vkconfig executable comes pre-built, be sure qt is installed to run it. Run vkconfig on the command line.
```

Comment:
, => ;
qt => Qt

It would be nice to provide instructions or a link to where Qt can be downloaded and installed, but maybe that's outside the scope of this document.

Reply:
I'm too lazy to want to go look up the package name.  I'd want qt5-default here but if we advance to qt6 we'd have to change the docs. How much of a problem is that?